src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/entities/ccd/HearingCentre.java

```diff
@@ -18,7 +18,8 @@
     NORTH_SHIELDS("northShields", "North Shields"),
     NOTTINGHAM("nottingham", "Nottingham Justice Centre"),
     TAYLOR_HOUSE("taylorHouse", "Taylor House"),
-    NEWCASTLE("newcastle", "Newcastle Civil & Family Courts and Tribunals Centre");
+    NEWCASTLE("newcastle", "Newcastle Civil & Family Courts and Tribunals Centre"),
+    REMOTE_HEARING("remoteHearing","Remote Hearing");
```

Comment:
Check the case. I think it needs to be `REMOTE_HEARING("remoteHearing","Remote hearing")` to be the same as what we've used in the other services.